modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/util/RelayUtils.java

```
@@ -236,6 +238,39 @@ public static boolean isDeleteRequestWithoutPayload (MessageContext msgContext)
         return false;
     }
 
+    private static boolean isEmptyPayloadStream(MessageContext messageContext) throws AxisFault {
+
+        boolean isEmpty = false;
+        BufferedInputStream bufferedInputStream = (BufferedInputStream) messageContext
+                .getProperty(PassThroughConstants.BUFFERED_INPUT_STREAM);
+        if (bufferedInputStream != null) {
+            try {
+                bufferedInputStream.reset();
+                bufferedInputStream.mark(0);
+            } catch (Exception e) {
+                // just ignore the error
```

Why are we ignoring the exception here?